ykrt/src/mt.rs

```diff
@@ -188,6 +188,9 @@ pub struct MTThread {
     inner: Rc<MTThreadInner>,
 }
 
+impl !Send for MTThread {}
+impl !Sync for MTThread {}
```

Comment:
You could add doc comments explaining why it is not `Send`/`Sync`. You can add doc comments to any impl, even negative ones.